packages/ocean-react/src/CrossSellCard/CrossSellCard.tsx

```diff
@@ -39,10 +14,20 @@ const CrossSellCard = React.forwardRef<HTMLDivElement, CrossSellCardProps>(
       ctaText,
       ctaAction,
       customIcon,
+      backgroundColor,
       ...rest
     },
     ref
   ) {
+    const titleCard: string = typeof title === 'string' ? title : title['text'];
```

Comment:
Why the title would come as an object? It's simple to receive as an independent attribute.

Reply:
The right way to change a component color scheme from the design system is by establishing the component variations then all the corresponding attributes will if necessary change colors automatically. That leaves no needs to receive an object with `text` and `color`.